google/cloud/spanner/integration_tests/backup_integration_test.cc

```diff
@@ -424,10 +448,15 @@ TEST_F(BackupTest, BackupTestWithCMEK) {
   auto encryption_config = CustomerManagedEncryption(encryption_key);
 
   Database db(in, spanner_testing::RandomDatabaseName(generator_));
-  auto database =
-      database_admin_client_
-          .CreateDatabase(db, /*extra_statements=*/{}, encryption_config)
-          .get();
+  auto pending_create = database_admin_client_.CreateDatabase(
+      db, /*extra_statements=*/{}, encryption_config);
+
+  // TODO(#4306) - this test times out (rarely) additional logs to troubleshoot
+  if (pending_create.wait_for(std::chrono::minutes(5)) != fs::ready) {
+    GCP_LOG(WARNING) << "Potential timeout in CreateDatabase()" << in;
+    google::cloud::LogSink::Instance().Flush();
+  }
+  auto database = pending_create.get();
```

Comment:
Have we ever seen `CreateDatabase()` taking a long time?

Reply:
Ooops, I changed my mind about what needed extra logging and did not revert this change. Fixed.